src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -177,7 +184,23 @@ private static void init(){}
 
         private static TagKey<Block> tag(String name)
         {
-            return BlockTags.create(new ResourceLocation("forge", name));
+            return BlockTags.create(new ResourceLocation("common", name));
+        }
+    }
+
+    public static class BlockEntityTypes
+    {
+        private static void init() {}
+
+        /**
+         * Blocks should be included in this tag if their movement can cause serious issues such as world corruption
+         * upon being moved, such as chunk loaders or pipes, for mods that move block entities.
+         */
+        public static final TagKey<BlockEntityType<?>> RELOCATION_NOT_SUPPORTED = tag("relocation_not_supported");
```

Comment:
What is the difference between this and the block tag? Are mods expected to check both before relocating, or am I required to tag both my block and my BE?

I would recommend having the java docs link to each other.

Reply:
I'll add javadoc (good catch there) but some mods might have multiple block entities based on the same block but not every BE is relocatable. And someone could have a non-BE block that isn't relocatable. Yeah this means the checker has to check block tag first. Then see if it is a BE, then check BE tag next for maximum compat with how other mods might had setup their blocks. I did find there is usage right now of both a block and block entity type tag so definitely linking the two for better visibility is the way to go. 
![image](https://github.com/neoforged/NeoForge/assets/40846040/454b29b7-e15d-4792-ae6c-edcaef86061b)
![image](https://github.com/neoforged/NeoForge/assets/40846040/c47637e3-1c96-4a04-a6bb-ccb81a9936a7)